packages/core/data/oot/world_mq/shadow_temple_mq.yml

```diff
@@ -102,6 +114,10 @@
     "MQ Shadow Temple After Wind Bomb Chest": "has_explosives"
     "MQ Shadow Temple GS Wind Hint": "true"
     "MQ Shadow Temple GS After Wind Bomb": "has_explosives"
+    "MQ Shadow Temple Pot Room Before Boat 1": "(has_weapon || can_use_sticks || can_use_din) || (can_play_sun)" #It would be too unfair to get around the Gibdos otherwise
+    "MQ Shadow Temple Pot Room Before Boat 2": "(has_weapon || can_use_sticks || can_use_din) || (can_play_sun)" #It would be too unfair to get around the Gibdos otherwise
```

Comment:
these two pots don't need the parenthesis.

Reply:
fixed.